elixir/apps/domain/test/support/fixtures/policies_fixtures.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Domain.PoliciesFixtures do
+  alias Domain.{AccountsFixtures, ActorsFixtures, AuthFixtures, ResourcesFixtures}
+
+  def policy_attrs(attrs \\ %{}) do
+    name = "policy-#{counter()}"
+
+    Enum.into(attrs, %{
+      name: name,
+      actor_group_id: nil,
+      resource_id: nil
+    })
+  end
+
+  def create_policy(attrs \\ %{}) do
+    attrs = policy_attrs(attrs)
+
+    {account, attrs} =
+      Map.pop_lazy(attrs, :account, fn ->
+        AccountsFixtures.create_account()
+      end)
+
+    {subject, attrs} =
+      Map.pop_lazy(attrs, :subject, fn ->
+        actor = ActorsFixtures.create_actor(type: :account_admin_user, account: account)
+        identity = AuthFixtures.create_identity(account: account, actor: actor)
+        AuthFixtures.create_subject(identity)
+      end)
+
+    {actor_group_id, attrs} =
+      Map.pop_lazy(attrs, :actor_group, fn ->
+        actor = ActorsFixtures.create_group(account: account, subject: subject)
+        actor.id
+      end)
+
+    {resource_id, attrs} =
+      Map.pop_lazy(attrs, :resource, fn ->
+        resource = ResourcesFixtures.create_resource(account: account, subject: subject)
+        resource.id
+      end)
+
+    {:ok, policy} =
+      attrs
+      |> Map.put(:actor_group_id, actor_group_id)
+      |> Map.put(:resource_id, resource_id)
+      |> Domain.Policies.create_policy(subject)
+
+    policy
+  end
+
+  # def create_resource(attrs \\ %{}) do
```

Comment:
This should be removed

Reply:
Ah, sorry about that, thought I had removed all commented out stuff.  Will get this removed.

Reply:
Done